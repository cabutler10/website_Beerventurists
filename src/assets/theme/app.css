html {
  height: 100%;
}

body {
  margin: 0;
  padding: 0;
  font-family: 'Open Sans', sans-serif;
  background-color: #ffffff;
  overflow-x: hidden;
  font-size: 14px;
  width: '100%';
}

a {
  text-decoration: none;
}
