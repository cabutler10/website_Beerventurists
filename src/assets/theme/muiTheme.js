import { createMuiTheme } from '@material-ui/core/styles'
import 'typeface-open-sans'
import 'typeface-permanent-marker'
import './app.css'

const status = {
  white: '#ffffff',
  black: '#040001',
  greyLt: '#F7F5F5',
  greyMed: '#656565',
  grey: '#c5c6c7',
  greyDk: '#404040',
  green: '#00942b',
  yellow: '#fce566',
  red: '#FF0012',
  blue: '#015b99',
  blueLt: '#0172c0',
  blueDk: '#014472',
  orange: '#c56113',
  orangeDk: '#c56113',
  orangeLt: '#ebab2f',
  tan: '#FFFBEF',
}

const theme = createMuiTheme({
  palette: {
    type: 'light',
    primary: {
      light: status.blueLt,
      main: status.blue,
      dark: status.blueDk,
    },
    secondary: {
      light: status.orangeLt,
      main: status.orange,
      dark: status.orangeDk,
    },
    error: { main: status.red },
  },
  status,
  typography: {
    useNextVariants: true,
    fontFamily: "'Open Sans', sans-serif",
    h5: {
      letterSpacing: 3,
    },
    h2: {
      fontFamily: "'Permanent Marker', cursive",
    },
    body1: {
      letterSpacing: 1,
      lineHeight: 1.8,
      //TODO is this an error in MUI? not sure why its default is 16 when MUI default is 14
      fontSize: 14,
    },
    body2: {
      letterSpacing: 1,
      fontSize: 14,
    },
    button: {
      letterSpacing: 3,
    },
  },
  overrides: {
    MuiTypography: {
      colorPrimary: {
        color: status.black,
      },
      colorSecondary: {
        color: status.orangeLt,
      },
      colorTextSecondary: {
        color: status.white,
      },
    },
    MuiButton: {
      root: {
        padding: '9px 27px',
        borderRadius: 0,
      },
      contained: {
        boxShadow: 'none',
      },
    },
  },
})

export default theme
