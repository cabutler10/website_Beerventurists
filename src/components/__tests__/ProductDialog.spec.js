import React from 'react'
import { render } from 'react-testing-library'
import ProductDialog from '../ProductDialog'
import { ThemeProvider } from '@material-ui/styles'
import theme from '../../assets/theme/muiTheme'

describe('ProductDialog', () => {
  it('renders correctly', () => {
    const mockProps = {
      handleDialogOpen: jest.fn,
      handleDialogClose: jest.fn,
      isDialogOpen: true,
      product: {
        images: [],
        more: { en: [] },
      },
      buttonText: 'mockButtonText',
      t: () => '',
    }
    const component = render(
      <ThemeProvider theme={theme}>
        <ProductDialog {...mockProps} />
      </ThemeProvider>
    )
    expect(component).toMatchSnapshot()
  })
})
