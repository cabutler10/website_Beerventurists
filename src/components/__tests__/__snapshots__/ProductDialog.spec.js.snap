// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductDialog renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body
    style="overflow: hidden; padding-right: 0px;"
  >
    <div
      aria-hidden="true"
    >
      <button
        class="MuiButtonBase-root-27 MuiButton-root-1 MuiButton-outlined-9 MuiButton-outlinedPrimary-10 Hook-button-lupj5y"
        tabindex="0"
        type="button"
      >
        <span
          class="MuiButton-label-2"
        >
          mockButtonText
        </span>
        <span
          class="MuiTouchRipple-root-46"
        />
      </button>
    </div>
    <div
      class="MuiModal-root-44 MuiDialog-root-30"
      role="dialog"
    >
      <div
        aria-hidden="true"
        class="MuiBackdrop-root-53"
        style="opacity: 1; -webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
      />
      <div
        class="MuiDialog-container-33 MuiDialog-scrollPaper-31"
        role="document"
        style="opacity: 1; -webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
        tabindex="-1"
      >
        <div
          class="MuiPaper-root-55 MuiPaper-elevation24-81 MuiPaper-rounded-56 MuiDialog-paper-34 Hook-paper-y3zfpz MuiDialog-paperScrollPaper-35 MuiDialog-paperWidthMd-39"
        >
          <div
            class="MuiDialogTitle-root-82 Hook-dialogTitle-1w0wwyx"
            id="dialog-title"
          >
            <h2
              class="MuiTypography-root-83 MuiTypography-title-89"
            >
              <button
                aria-label="close"
                class="MuiButtonBase-root-27 MuiIconButton-root-119 MuiIconButton-colorInherit-120 Hook-closeButton-112e1qq"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiIconButton-label-124"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root-125"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                    />
                    <path
                      d="M0 0h24v24H0z"
                      fill="none"
                    />
                  </svg>
                </span>
                <span
                  class="MuiTouchRipple-root-46"
                />
              </button>
            </h2>
          </div>
          <div
            class="MuiDialogContent-root-134 Hook-dialogRoot-1xsx80b"
          >
            <div
              class="Hook-dialogContainer-102dr8y"
            >
              <div
                class="Hook-dialogImg-tyu7fo"
              >
                <div
                  style="overflow-x: hidden;"
                >
                  <div
                    class="react-swipeable-view-container"
                    style="flex-direction: row; direction: ltr; display: flex;"
                  />
                </div>
                <div
                  class="MuiPaper-root-55 MuiPaper-elevation0-57 MuiMobileStepper-root-135 MuiMobileStepper-positionStatic-138 Hook-mobileStepper-1equy9r"
                >
                  <button
                    aria-label="scroll right"
                    class="MuiButtonBase-root-27 MuiButtonBase-disabled-28 MuiIconButton-root-119 MuiIconButton-disabled-123"
                    disabled=""
                    tabindex="-1"
                    type="button"
                  >
                    <span
                      class="MuiIconButton-label-124"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root-125"
                        focusable="false"
                        role="presentation"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M15.41 16.59L10.83 12l4.58-4.59L14 6l-6 6 6 6 1.41-1.41z"
                        />
                        <path
                          d="M0 0h24v24H0V0z"
                          fill="none"
                        />
                      </svg>
                    </span>
                  </button>
                  <div
                    class="MuiMobileStepper-dots-139"
                  />
                  <button
                    aria-label="scroll left"
                    class="MuiButtonBase-root-27 MuiIconButton-root-119"
                    tabindex="0"
                    type="button"
                  >
                    <span
                      class="MuiIconButton-label-124"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root-125"
                        focusable="false"
                        role="presentation"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
                        />
                        <path
                          d="M0 0h24v24H0V0z"
                          fill="none"
                        />
                      </svg>
                    </span>
                    <span
                      class="MuiTouchRipple-root-46"
                    />
                  </button>
                </div>
              </div>
              <div
                class="Hook-dialogContent-ikr7rs"
              >
                <div
                  class="Hook-priceContainer-19tfvb6"
                >
                  <h6
                    class="MuiTypography-root-83 MuiTypography-h6-100 Hook-price-1gw20eh"
                  >
                    â‚¬undefined
                  </h6>
                  <div />
                </div>
                <ul
                  class="MuiList-root-143 Hook-scroll-y6j20v"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div
    aria-hidden="true"
  >
    <button
      class="MuiButtonBase-root-27 MuiButton-root-1 MuiButton-outlined-9 MuiButton-outlinedPrimary-10 Hook-button-lupj5y"
      tabindex="0"
      type="button"
    >
      <span
        class="MuiButton-label-2"
      >
        mockButtonText
      </span>
      <span
        class="MuiTouchRipple-root-46"
      />
    </button>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllBySelectText": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getAllByValue": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getBySelectText": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "getByValue": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllBySelectText": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryAllByValue": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryBySelectText": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "queryByValue": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
